akka-docs/src/main/paradox/scala/remoting-artery.md

```diff
@@ -317,9 +363,19 @@ Messages sent with actor selection are by default discarded in untrusted mode, b
 permission to receive actor selection messages can be granted to specific actors
 defined in configuration:
 
-```
-akka.remote.artery..trusted-selection-paths = ["/user/receptionist", "/user/namingService"]
-```
+Scala
+:   @@@vars
+    ```
+    akka.remote.artery.trusted-selection-paths = ["/user/receptionist", "/user/namingService"]
+    ```
+    @@@
+
+Java
+:   @@@vars
+    ```
+    akka.remote.artery.trusted-selection-paths = ["/user/receptionist", "/user/namingService"]
+    ```
+    @@@
```

Comment:
Looks like this is the same for Java and Scala - perhaps make it just a normal snippet without tabs?